.github/workflows/python.yml

```diff
@@ -109,24 +91,63 @@ jobs:
               run: |
                   source .env/bin/activate
                   cd python/tests/
-                  pytest --asyncio-mode=auto --override-ini=addopts= --load-module=$GITHUB_WORKSPACE/redisjson.so
+                  pytest --asyncio-mode=auto --override-ini=addopts= --load-module=$GITHUB_WORKSPACE/redisjson.so --html=pytest_report.html --self-contained-html
 
             - uses: ./.github/workflows/test-benchmark
               with:
                   language-flag: -python
 
-    lint-rust:
+            - name: Upload test reports
+              if: always()
+              continue-on-error: true
+              uses: actions/upload-artifact@v4
+              with:
+                  name: test-reports-python-redis-${{ matrix.redis }}
+                  path: |
+                      python/pytest_report.html
+                      utils/clusters/**
+                      benchmarks/results/**
+
+    lint:
         runs-on: ubuntu-latest
         timeout-minutes: 15
         steps:
             - uses: actions/checkout@v4
               with:
                   submodules: recursive
 
-            - uses: ./.github/workflows/lint-rust
+            - name: lint rust
+              uses: ./.github/workflows/lint-rust
               with:
                   cargo-toml-folder: ./python
-              name: lint python-rust
+
+            - name: Install dependencies
+              if: always()
+              working-directory: ./python
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install flake8 isort black
+
+            - name: Lint python with isort
+              if: always()
+              working-directory: ./python
+              run: |
+                  isort . --profile black --check --diff
+
+            - name: Lint python with flake8
+              if: always()
+              working-directory: ./python
+              run: |
+                  # stop the build if there are Python syntax errors or undefined names
+                  flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --extend-ignore=E230 --exclude=python/glide/protobuf,.env/*
+                  # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+                  flake8 . --count --exit-zero --max-complexity=12 --max-line-length=127 --statistics --extend-ignore=E230 --exclude=python/glide/protobuf,.env/*
```

Comment:
`--exit-zero` suppresses all errors/warnings - this linter prints reports read by nobody.
Should I enforce that lint?

Reply:
This seems pretty intentional, based on the comment. I'd ask whoever put this in here if it's okay to enforce now.